config/v3_2_experimental/schema/ignition.json

```diff
@@ -514,6 +514,9 @@
             },
             "shell": {
               "type": ["string", "null"]
+            },
+            "shouldExist": {
```

Comment:
The convention is to use `must` over `should`.

Reply:
Ignition already uses `shouldExist` for [Partitions](https://github.com/coreos/ignition/blob/master/config/v3_2_experimental/types/schema.go#L115).